boards/arm/nrf52840_mdk/Kconfig

```diff
@@ -3,8 +3,19 @@
 # Copyright (c) 2018 makerdiary.com
 # SPDX-License-Identifier: Apache-2.0
 
+if BOARD_NRF52840_MDK
+
 config BOARD_ENABLE_DCDC
 	bool "Enable DCDC mode"
 	select SOC_DCDC_NRF52X
 	default y
-	depends on BOARD_NRF52840_MDK
+
+config BOARD_HAS_NRF5_BOOTLOADER
```

Comment:
There seems to be a list of supported bootloaders in Kconfig.zephyr, where we could support the nRF5 Bootloader. This would be a more generic solution, not coupled with the _mdk board.